### DISCUSSION TITLE
2006/06/27/tiobe-2006%E5%B9%B46%E6%9C%88%E6%9C%80%E6%B5%81%E8%A1%8C%E5%BC%80%E5%8F%91%E8%AF%AD%E8%A8%80%E6%8E%92%E5%90%8D

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#88] [Announcements]
# 2006/06/27/tiobe-2006%E5%B9%B46%E6%9C%88%E6%9C%80%E6%B5%81%E8%A1%8C%E5%BC%80%E5%8F%91%E8%AF%AD%E8%A8%80%E6%8E%92%E5%90%8D

TIOBE 语言排名一个月更新一次。它的统计方法是基于全球有经验的工程师，科目和第三方厂商. 同时也参考来自google,msn,yahoo等搜索引擎计算得出的结果。当然，这个排名并不说明哪个语言绝对最好，但是它可以给你一个参考，当你构建一个新的软件系统时，应该采用什么样的语言。

https://blog.qizhen.xyz/2006/06/27/tiobe-2006%E5%B9%B46%E6%9C%88%E6%9C%80%E6%B5%81%E8%A1%8C%E5%BC%80%E5%8F%91%E8%AF%AD%E8%A8%80%E6%8E%92%E5%90%8D

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Loren
2007年06月8日 20:28 编辑
嘿，我最近在开发工程，用到LabView，以前也想学，但是没有合适的项目下锅，这回赶上了！嘎嘎

回复
Qizhen
2006年07月20日 11:08 编辑
写书时非常累得一件事，我先写写blog练练笔，然后再考虑：）

回复
Tian
2006年07月4日 22:38 编辑
你写本LabVIEW的书吧，可以赚赚稿费，而且有机会成为有史以来最好的LabVIEW编程书哦。

回复
Wu
2006年06月28日 10:15 编辑
排名不说明问题。合适的才是最好的。工具好，还要用的好。打球输了，不能怪拍子，是一样的道理 :-)

回复
